bindings/c/include/manifoldc.h

```diff
@@ -188,6 +188,9 @@ ManifoldManifold *manifold_set_properties(
     void (*fun)(float *new_prop, ManifoldVec3 position, const float *old_prop));
 ManifoldManifold *manifold_calculate_curvature(void *mem, ManifoldManifold *m,
                                                int gaussian_idx, int mean_idx);
+ManifoldManifold *manifold_calculate_normals(void *mem, ManifoldManifold *m,
```

Comment:
@geoffder I tried adding `manifold_smooth`, but you already have that for the static version and apparently overloads are not allowed?? I really hate C... Anyway, how would you like to resolve the naming?